MH4U Dex/RegionEncylcopediaCollectionViewCell.m

```diff
@@ -8,6 +8,26 @@
 
 #import "RegionEncylcopediaCollectionViewCell.h"
 
+@interface RegionEncylcopediaCollectionViewCell ()
+
+@property (nonatomic, weak) IBOutlet UILabel *regionNameLabel;
+@property (nonatomic, weak) IBOutlet UIImageView *regionIconImageView;
+
+@end
+
 @implementation RegionEncylcopediaCollectionViewCell
 
+- (void)displayContents
+{
+    NSString *imageName = [NSString stringWithFormat:@"%@-icon", self.regionKeyName];
+    UIImage *image = [UIImage imageNamed:imageName];
+    if (image) {
+        self.regionIconImageView.image = image;
+    } else {
+        self.regionIconImageView.image = nil;
+    }
```

Comment:
The `else` clause here is basically "If `image` is `nil`, set `self.regionIconImageView.image` to `nil`", which could just as easily be setting `self.regionIconImageView.image` to `image`—so you don't really need the `if`-`else`, unless there's something else coming here?